test/integration/local/test_pre_post_processing_mme1.py

```diff
@@ -27,27 +26,27 @@
 
```

Comment:
Shall we add comments in each of the 4 mme tests to describe the test scenario?

Reply:
Added the comments in each of the mme tests to describe the scenario.